### PR TITLE
Added interview question model and repository

### PR DESCRIPTION
**Pull request by TomasNiemczyk98**
## Summary

Added interview question model and repository

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactoring (non-breaking change to code and structure)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/WebApi/Application/Repositories/IInterviewQuestionRepository.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿using Domain.Entities;
+using Domain.Models;
+
+namespace Application.Repositories
```
Comment (PlesnikJakub) [inline src/WebApi/Application/Repositories/IInterviewQuestionRepository.cs:4]: Let's convert all namespaces to the file scoped version. We will enforce it as a standard across whole repository. 

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces

File: src/WebApi/Domain/Models/InterviewQuestionModel.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿namespace Domain.Models
+{
+    public class InterviewQuestionModel
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
```
Comment (PlesnikJakub) [inline src/WebApi/Domain/Models/InterviewQuestionModel.cs:6]: Please fix formatting and add newlines between props so its consistent between Model and Entity class 😊

**Review by PlesnikJakub — CHANGES_REQUESTED** [review]
There are small changes required. Basically, it's just namespaces and formatting, other than that it looks good 😊 Please make those small changes then we can ship it.

**Review by PlesnikJakub — APPROVED** [review]
LGTMT 🚀

**Comment by petrspelos** [comment]
@TomasNiemczyk98 feel free to delete the feature branch if it's not needed. ☺️

**Pull request merged**